boards/ssrc/icicle/nuttx-config/scripts/script.ld

```diff
@@ -20,16 +20,15 @@
 
 MEMORY
 {
-  progmem (rx) : ORIGIN = 0xAFB00000, LENGTH = 3072K  /* w/ cache */
+  toc     (rx) : ORIGIN = 0xAFB00000. LENGTH = 64 /* w/ cache */
```

Comment:
Nope, this size is simply not enough. #define MAX_TOC_ENTRIES 32, one entry is 32 octets. So 1K + a few octets extra is needed.

Back to the drawing board I guess..